sky/check.py

```diff
@@ -1,26 +1,29 @@
 """Credential checks: check cloud credentials and enable clouds."""
 import traceback
-from typing import Dict, Iterable, List, Optional, Tuple
+from typing import Any, Dict, Iterable, List, Optional, Tuple
 
 import click
 import colorama
 import rich
 
-from sky import clouds
+from sky import clouds as sky_clouds
 from sky import exceptions
 from sky import global_user_state
 from sky.adaptors import cloudflare
 from sky.utils import ux_utils
 
 
-# TODO(zhwu): add check for a single cloud to improve performance
-def check(quiet: bool = False, verbose: bool = False) -> None:
+def check(
+    quiet: bool = False,
+    verbose: bool = False,
+    clouds: Optional[Tuple[str]] = None,
```

Comment:
nit: how about renaming this to `clouds_to_check` and import as original? Reason: to reduce confusion compared to other files in the system

Reply:
I think it might be good to keep the CLI args clean - e.g., `Usage: sky check [OPTIONS] [CLOUDS]...` instead of `Usage: sky check [OPTIONS] [CLOUDS_TO_CHECK]`. Let me know if you feel strongly about this and I can change it

Reply:
Oh that makes sense. Does not notice that. Lets keep the current implementations ;)